Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
```

Comment:
Some rust code?

Reply:
Uh... that's not meant to be there 😆

Reply:
Fixed in a2296931. This is a mess of mine.